lib/ecto/query/builder/select.ex

```diff
@@ -34,7 +34,7 @@ defmodule Ecto.Query.Builder.Select do
   def escape(other, vars, env) do
     cond do
       take?(other) ->
-        {{:{}, [], [:&, [], [0]]}, {[], %{0 => {:any, other}}}}
+        {{:{}, [], [:&, [], [0]]}, {[], %{0 => {:any, Macro.expand(other, env)}}}}
```

Comment:
I think this macro expansion may be not necessary, but I like the idea of expanding the sigil here instead of passing it on. wdyt?